src/VirtualClient/VirtualClient.Actions.UnitTests/SPEC/SpecViewExecutorTests.cs

```diff
@@ -123,6 +123,11 @@ private async Task SpecViewExecutorExecutesWorkloadAsExpected(PlatformID platfor
                         return this.mockFixture.Process;
                     };
 
+                    // Remove any mock blob managers so that we do not evaluate the code paths that
+                    // upload log files by default.
+                    this.mockFixture.Dependencies.RemoveAll<IEnumerable<IBlobManager>>();
```

Comment:
The tabbing seems odd

Reply:
I left this line inside `if (platform == PlatformID.Win32NT)` as we only suporrt win-x64 for now.